### PR TITLE
dessert are funny

### PR DESCRIPTION
**Pull request by DCorda343**
Roast me. Pun intended. 

txt file with desserts, i think.

### CODE REVIEW

File: desserts.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Canole 
+Pie
+$$$$$
+3.1415
```
Comment (coxas) [inline desserts.txt:4]: how do you eat this

Comment (DCorda343) [reply desserts.txt:4]: In my native homeland of Lilliputa, we have 3.1415 for breakfast.

Comment (coxas) [reply desserts.txt:4]: Not 3.14159? Precision tastes good

**Review by DCorda343 — COMMENTED** [review]
I forgot!

**Pull request merged**